earth_enterprise/docs/landing_page/manual/answer/2981872.html

```diff
@@ -56,9 +56,6 @@
 </p></li>
 geselectassetroot [--lock] [--noprompt] [--unlock]
```

Comment:
Not directly related to this change, but these two lines should probably be in a <code> tag.